test/parallel/test-os.js

```diff
@@ -191,3 +181,20 @@ is.string(pwd.username);
 assert.ok(pwd.homedir.includes(path.sep));
 assert.strictEqual(pwd.username, pwdBuf.username.toString('utf8'));
 assert.strictEqual(pwd.homedir, pwdBuf.homedir.toString('utf8'));
+
+// Test that the Symbol.toPrimitive functions work correctly
+/* eslint-disable no-restricted-properties */
+[
+
+  [assert.equal, `${os.freemem}`, os.freemem()],
```

Comment:
> This was intentional, recognizing that the differences in the values should still be equal.

You might be misunderstanding my suggestion. I know it's intentional. Explicitly converting the number to a string and using `strictEqual()` makes it more clear that it's intentional, at least in my opinion. OK by me, though, if you prefer the way it is.